docs/flexbox.md

```diff
@@ -63,13 +106,29 @@ export default class JustifyContentBasics extends Component {
 // skip this line if using Create React Native App
 AppRegistry.registerComponent('AwesomeProject', () => JustifyContentBasics);
 ```
+See some examples of [wix](https://medium.com/wix-engineering/the-full-react-native-layout-cheat-sheet-a4147802405c) engineers
+![Justify Content](https://cdn-images-1.medium.com/max/800/1*i5TVlme-TisAVvD5ax2yPA.png)
 
 #### Align Items
 
+[`alignItems`](https://facebook.github.io/react-native/docs/layout-props#alignitems) describes how to align children along the cross axis of their container. Align items is very similar to `justifyContent` but instead of applying to the main axis, `align items` applies to the cross axis.
```

Comment:
```suggestion
[`alignItems`](https://facebook.github.io/react-native/docs/layout-props#alignitems) describes how to align children along the cross axis of their container. Align items is very similar to `justifyContent` but instead of applying to the main axis, `alignItems` applies to the cross axis.
```

- Fix variable